app/jobs/reports/sp_user_counts_report.rb

```diff
@@ -15,7 +15,60 @@ def perform(_date)
       user_counts = transaction_with_timeout do
         Db::Identity::SpUserCounts.call
       end
-      save_report(REPORT_NAME, user_counts.to_json, extension: 'json')
+
+      track_report_data_events(user_counts)
+      results = save_report(REPORT_NAME, user_counts.to_json, extension: 'json')
+
+      track_global_user_total_events
+      results
+    end
+
+    private
+
+    def track_report_data_events(user_counts)
+      user_counts.each do |hash|
+        track_report_data_event(
+          Analytics::REPORT_SP_USER_COUNTS,
+          issuer: hash['issuer'],
+          user_total: hash['total'],
+          ial1_user_total: hash['ial1_total'],
+          ial2_user_total: hash['ial2_total'],
+          app_id: hash['app_id'].to_s,
+        )
+      end
+    end
+
+    def track_global_user_total_events
+      track_global_registered_users
+      track_ial1_users_linked_to_sps
+      track_ial2_users_linked_to_sps
```

Comment:
see other comment about DRY

```suggestion
      track_users_linked_to_sps(ial: 1, event: Analytics::REPORT_IAL1_USERS_LINKED_TO_SPS_COUNT)
      track_users_linked_to_sps(ial: 2, event: Analytics::REPORT_IAL2_USERS_LINKED_TO_SPS_COUNT)
```